src/load.rs

```diff
@@ -24,13 +27,42 @@ pub fn new<I: Interfaceable>(
 
     let name = src.prompt();
     game.set_name(name.clone());
+    return write_protobuf(src, game);
+}
 
-    src.print(&format!(
-        "Hello {}, please enter the name of the new save file:",
-        &name
-    ));
-    let path = src.prompt();
+pub fn save<I: Interfaceable>(
+    src: &mut I,
+    game: &Game,
+) -> File {
 
-    File::create(&Path::new(&path)).unwrap();
-    return Ok(game);
+    let mut path;
+    let mut file;
```

Comment:
warning: variable does not need to be mutable
  --> src/load.rs:39:9
   |
39 |     let mut file;
   |         ---^^^^^
   |         |
   |         help: remove this `mut`